src/libraries/Microsoft.PowerFx.Core/Public/Config/FormulaValueSerializerSettings.cs

```diff
@@ -12,5 +12,7 @@ namespace Microsoft.PowerFx
     /// </summary>
     public class FormulaValueSerializerSettings
     {
+        // Provides a simpler version for some types.
```

Comment:
>// P [](http://example.com/codeflow?start=8&length=4)

super nit: xml comment. 
Add (we want to explain why this field isn't always set to True) 

Provides a simpler version for some types - this is more human readable, but may not be sufficient to fully deserialize the value back to its original